src/hooks/useVenuePoll.ts

```diff
@@ -0,0 +1,60 @@
+import { useMemo, useCallback } from "react";
+import firebase from "firebase/app";
+
+import { voteInVenuePoll, createVenuePoll, deleteVenuePoll } from "api/poll";
+
+import { PollMessage, PollValues, Vote } from "types/chat";
+
+import { useVenueId } from "./useVenueId";
+import { useUser } from "./useUser";
+
+export const useVenuePoll = () => {
+  const venueId = useVenueId();
+  const { user } = useUser();
```

Comment:
userid

Reply:
fixed in f2fa24aadcae7547799e463bd593e8a7a2cf05b5